sdk/include/opentelemetry/sdk/metrics/aggregator/sketch_aggregator.h

```diff
@@ -72,7 +72,7 @@ class SketchAggregator final : public Aggregator<T>
     }
     else
     {
-      idx = ceil(log(val) / log(gamma));
+      idx = static_cast<int>(ceil(log(val) / log(gamma)));
```

Comment:
I'm not too familiar with this part of the code, but shouldn't `idx` be a `size_t`? As we use it as an index below.

Reply:
I think `size_t` should be ideal but that's a different issue here. The issue here is `double` is returned from `ceil()`. I'll revisit the index type later.